### ISSUE TITLE
Missing assets 

### ISSUE THREAD

**Issue by wolfy01** [psi-4ward/psitransfer#86]
app/common.js
app/upload.js

**Comment by draft1** [comment]
Same here, no update on this project anymore?

**Comment by RemyyB** [comment]
Same issue here

**Comment by psi-4ward** [comment]
So you should give me some more information about how you installed and run PsiTransfer.

**Comment by RemyyB** [comment]
Tried to install it manually, precompiled and via Docker, all have the same issue, just the exact way of the examples on the readme

**Comment by psi-4ward** [comment]
```
$ docker run -p 0.0.0.0:3000:3000 psitrax/psitransfer
PsiTransfer listening on http://0.0.0.0:3000
2020-04-14T08:31:08.738Z GET / 200 6.519 172.17.0.1
2020-04-14T08:31:08.787Z GET /assets/styles.css 200 4.685 172.17.0.1
2020-04-14T08:31:08.796Z GET /app/upload.js 200 3.284 172.17.0.1
2020-04-14T08:31:08.802Z GET /app/common.js 200 3.349 172.17.0.1
2020-04-14T08:31:09.014Z GET /config.json 200 1.646 172.17.0.1
2020-04-14T08:31:09.147Z GET /assets/favicon.ico 200 1.764 172.17.0.1
2020-04-14T08:31:10.628Z GET /assets/styles.css 304 1.221 172.17.0.1
```

so the docker container looks good for me.
But you're right, the release-version has a problem, I'll fix it.

**Comment by psi-4ward** [comment]
Released 1.3.4 which was build by travis so this shouldn't happen again.
